src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using SFA.DAS.ApplyService.Domain.Apply;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ApplyService.Web.ViewModels.Roatp
+{
+    public class TwoInTwelveMonthsViewModel : IPageViewModel
+    {
+        public bool? HasTwoInTwelveMonths { get; set; }
+
+        public string Title { get { return "Have you had 2 unsuccessful applications in the last 12 months?"; } set { } }
```

Comment:
does this need a set? If not, => will do

Reply:
abstract base class so needs an implementation :rage1: